Assets/Scripts/Fire.cs

```diff
@@ -0,0 +1,50 @@
+﻿    using System.Collections;
+using System.Collections.Generic;
+using UnityEditor;
+using UnityEngine;
+
+public class Fire : MonoBehaviour
```

Comment:
This is more a matter of style but I think normally brackets are on the same line in C#. I kinda prefer it that way but if you have a strong preference of using a newline let me know, then we can battle it out :p. Just as long as we are consistent it is fine.

I also won't make another comment for each bracket in this script